azurecontainerapps.ts

```diff
@@ -150,8 +154,12 @@ export class azurecontainerapps {
         // Get the YAML configuration file, if provided
         this.yamlConfigPath = this.toolHelper.getInput('yamlConfigPath', false) as string;
 
+        // Get the name of the image to build, if provided
+        this.imageToBuild = this.toolHelper.getInput('imageToBuild', false) as string;
```

Comment:
Aren't we already reading the value for this input elsewhere in the code?

Reply:
We don't need this anymore because we removed the validations for `imageToBuild` arg.